slackbot/dispatcher.py

```diff
@@ -98,7 +98,7 @@ def _default_reply(self, msg):
         default_reply = [
             u'Bad command "%s", You can ask me one of the following questions:\n' % msg['text'],
         ]
-        default_reply += [u'    • `%s`' % str(p.pattern) for p in self._plugins.commands['respond_to'].iterkeys()]
+        default_reply += [u'    • `%s`' % unicode(p.pattern) for p in self._plugins.commands['respond_to'].iterkeys()]
```

Comment:
I think there is a similar issue in line 119:

    text = '<@{}>: {}'.format(self._get_user_id(), text)

there the format string should have a `u` prefix.